docs/topics/cross-platform-mobile-development.md

```diff
@@ -72,21 +76,23 @@ There are plenty of reasons businesses choose this approach over other options.
 
 With cross-platform programming, mobile engineers don't need to write new code for every operating system. Using a single codebase allows developers to cut down on time spent doing repetitive tasks, such as API calls, data storage, data serialization, and analytics implementation.
 
-In our Kotlin Multiplatform survey from Q3-Q4 2021, we asked the Kotlin community about the parts of code they were able to share between different platforms.
+Technologies like Kotlin Multiplatform allow you to implement the data, business, and presentation layers of your app just once. Alternatively, you can adopt KMP  gradually: Choose a piece of logic that changes frequently and usually goes out of sync, such as data validation, filtering, or sorting; make it cross-platform; and then connect it to your project as a microlibrary.
+
+At JetBrains, we regularly conduct Kotlin Multiplatform surveys and ask our community members about what parts of code they share between different platforms.
 
-![Parts of code Kotlin Multiplatform Mobile users can share between platforms](survey-results-q1-q2-22.png){width=700}
+![Parts of code Kotlin Multiplatform users can share between platforms](survey-results-q1-q2-22.png){width=700}
 
 #### 2. Time savings
 
-Due to code reusability, cross-platform applications require less code, and when it comes to coding, less code is more. Time saved is because you do not have to write as much code. Additionally, with fewer lines of code, there are fewer places for bugs to emerge, resulting in less time spent testing and maintaining your code.
+Due to code reusability, cross-platform applications require less code, and when it comes to coding, less code is more. Time is saved because you do not have to write as much code. Additionally, with fewer lines of code, there’s less room for bugs to emerge, resulting in less time spent testing and maintaining your code.
```

Comment:
```suggestion
Due to code reusability, cross-platform applications require less code, and when it comes to coding, less code is more. Time is saved because you do not have to write as much code. Additionally, with fewer lines of code, there's less room for bugs to emerge, resulting in less time spent testing and maintaining your code.
```